### PR TITLE
Peer5 hlsjs support

### PR DESCRIPTION
**Pull request by costa**
I'm not sure why those commits (by Oren) are included with this comparison. I believe I've just merged the 2.44-5-June branch in, so there should have been just my commits in the comparison.

### CODE REVIEW

**Comment by itaykinnrot** [comment]
@costa  please close and re-open on 2.45-3-July branch - make sure you merge it to your branch before opening the pull. thanks,

**Comment by kaltura-hooks** [comment]
Hi @costa,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

File: modules/Peer5/resources/mw.Peer5.js [hunk 1]
```diff
@@ -1,104 +1,62 @@
...
+            // peer5JsUrl: 'https://peer5.com/peer5.js?id=XXXXXX'
+            // peer5HlsjsPluginJsUrl: 'https://peer5.com/peer5.hlsjs.plugin.js'
+        },
+
+        setup: function() {
+            var that = this;
+            var config = {};
+            var configValid = true;
+
+            var tAnyway = setTimeout(setupCompleteCallback, setTimeout);
```
Comment (shacharz) [inline modules/Peer5/resources/mw.Peer5.js]: setTimeout supposed to be the config's setupTimeout ?

File: modules/Peer5/resources/mw.Peer5.js [hunk 2]
```diff
@@ -1,104 +1,62 @@
...
+
+            function setupCompleteCallback() {
+                if (tAnyway) {
+                    clearTimeout(tAnyway);
+                    tAnyway = null;
+				            that.initCompleteCallback();
```
Comment (shacharz) [inline modules/Peer5/resources/mw.Peer5.js]: allign

**Pull request closed without merging**